website_sale_require_legal/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.http import request
+from openerp.addons.website_sale.controllers.main import website_sale
+
+
+class RequireLegalTermsToCheckout(website_sale):
+    def _get_mandatory_billing_fields(self, *args, **kwargs):
+        """Require accepting legal terms for unauthenticated users."""
```

Comment:
How do you legally bind an unauthenticated user ?
I can take it for authenticated ones, but this case seems not useful